topo/common_test.go

```diff
@@ -32,14 +32,14 @@ var batageljZaversnikGraph = []set{
 	20: nil,
 }
 
-// set is an integer set.
-type set map[int]struct{}
+// intset is an integer set.
```

Comment:
Yeah, I was thinking about rolling that in, but I don't want to expose linksTo. set here is really an edge list entry set.